templates/books-index.html

```diff
@@ -0,0 +1,30 @@
+<!-- header -->
+<div class="jumbotron">
+  <div class="container">
```

Comment:
Is there a reason you have jumbotron and container, or is that just because you copy/pasted from another assignment?